backtrace-library/src/main/java/backtraceio/library/logger/BacktraceLogger.java

```diff
@@ -71,13 +71,20 @@ public static int e(String tag, String message) {
      * @return the number of bytes written
      */
     public static int e(String tag, String message, Throwable tr) {
-        if (BacktraceLogger.logLevel.ordinal() <= LogLevel.ERROR.ordinal()) {
-            return Log.e(getTag(tag), message, tr);
-        }
-        return 0;
+        return logger.e(tag, message, tr);
     }
 
-    private static String getTag(String tag) {
-        return BacktraceLogger.BASE_TAG + tag;
+    /**
+     * Set logging level from which all messages should be logged to the console
+     * @param level login level (debug, warn, error, off)
+     *
+     * @deprecated setting the logging level should be done directly in the passed custom logger
+     * implementation so as not to depend on the internal implementation of Backtrace internal
+     * LogLevel types.
+     *
+     */
+    @Deprecated
+    public static void setLevel(@NotNull LogLevel level) {
+        logger.setLevel(level);
```

Comment:
lets think about this twice

Since this is the interface that we want to deprecate anyway, I think the logger interface will have some deprecated API as well. 

My suggestion is: 

- can we remove setLevel method from the logger definition? 
- since this is deprecated and we want to remove it - maybe we can use log level in this class and then pass data to Logger? 

With this approach we have less fields to deprecate and less changes the customer needs to make

Reply:
please review one more time